CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Adds recursive functionality to all fsspec connectors
 * Adds generic --recursive ingest flag
+* Adds --ocr-languages to unstructured-ingest
```

Comment:
When should we append to an existing version rather than creating a new one?

Reply:
since 0.7.8 was released, the convention is to create a new `0.7.9-devX` Changelog section (if another PR hasn't already been merged to do that). so, please update CHANGELOG to begin with:

```
## 0.7.9-dev0

### Enhancements

* Adds --ocr-languages to unstructured-ingest

### Features

### Fixes

## 0.7.8
...
```

and of course update unstructured/__version__.py to match.